operations/research-and-development/engineering/integrations.md

```diff
@@ -89,6 +89,24 @@ Seeing an intermittent connection issue with Zoom on http://dkh-local.ngrok.io.
 - Staff members add the issues there are working on for the current sprint to https://github.com/orgs/mattermost/projects/1.
     - Putting an issue is always preferred over putting a PR on the board.
 
+## Documentation process
+
+**1% draft**
+
+This guide outlines the processes and workflows for requesting, updating, managing, and maintaining documentation for Mattermost-managed and community-managed apps, plugins, and integrations. This includes the branching strategy for the Gitbook repos, how to set up a new Gitbook, and the process for aligning documentation with releases.
+
+Integrations comprise Mattermost-managed plugins and community-managed integrations.
+
+- Mattermost-managed integrations are maintained by Mattermost staff and the documentation is written and maintained by the Mattermost Technical Writer.
+- Community-managed integrations are maintained by their creator, which includes documentation maintenance.
+
+Mattermost integrations documentation currently spans:
+
+- docs.mattermost.com: Outline of how integrations work with Mattermost and a list of integrations available.
+- Apps Marketplace: Content for specific apps, which is hosted in developers.mattermost.com.
```

Comment:
I don't get what you a referring to. The Marketplace doesn't has any docs in it. Are you referring to the Apps Framework?

Reply:
No, this was referring to future plans where content for apps would be displayed in the Marketplace and live in the developer-documentation repo. It's not applicable at this stage though.